### PR TITLE
Restore tests part  II

### PR DESCRIPTION
**Pull request by Seb-Solon**
* Fixed a small bug
* Add util function for fetching broks in tests
* Add brok fetching in contacts too

### CODE REVIEW

File: test/test_customs_on_service_hosgroups.py [hunk 1]
```diff
@@ -47,38 +47,40 @@
...
 class TestCustomsonservicehosgroups(AlignakTest):
 
     def setUp(self):
-        self.setup_with_file(['etc/alignak_customs_on_service_hosgroups.cfg'])
+        self.setup_with_file('cfg/cfg_customs_on_service_hosgroups.cfg')
+        self._sched = self.schedulers['scheduler-master'].sched
 
     # We look for 3 services: on defined as direct on 1 hosts, on other
     # on 2 hsots, and a last one on a hostgroup
     def test_check_for_custom_copy_on_serice_hostgroups(self):
```
Comment (mohierf) [inline test/test_customs_on_service_hosgroups.py]: Missing test description. Add a 
""" We look for 3 services: on defined as direct on 1 hosts, on other on 2 hosts, and a last one on a hostgroup
"""

Same comment for all the tests :wink:

File: test/test_contactdowntimes.py [hunk 2]
```diff
@@ -47,11 +47,17 @@
...
+from alignak_test import AlignakTest
 
 
 class TestContactDowntime(AlignakTest):
 
+    def setUp(self):
+        self.setup_with_file("cfg/cfg_default.cfg")
+        self._sched = self.schedulers['scheduler-master'].sched
+
     def test_contact_downtime(self):
```
Comment (mohierf) [inline test/test_contactdowntimes.py]: Add test doc...

File: test/test_exclude_services.py [hunk 3]
```diff
@@ -42,28 +42,26 @@
...
 
 
 class TestPropertyOverride(AlignakTest):
 
     def setUp(self):
-        self.setup_with_file(['etc/exclude_include_services.cfg'])
+        self.setup_with_file('cfg/cfg_exclude_include_services.cfg')
+        self._sched = self.schedulers['scheduler-master'].sched
 
     def test_exclude_services(self):
```
Comment (mohierf) [inline test/test_exclude_services.py]: Add test doc...

**Comment by Seb-Solon** [comment]
Fixed / squashed

**Pull request merged**